### ISSUE TITLE
Add course websites to website

### ISSUE THREAD

**Issue by jamiecmontgomery** [UCSB-MEDS/UCSB-MEDS.github.io#20]
Hi @allisonhorst -

Could you link your course websites on the [Courses page](https://ucsb-meds.github.io/courses.html)? I think these need to be done in the googlesheet you have the course information in, and I am having issues linking googledrive with R on the server.

Thanks!

**Comment by allisonhorst** [comment]
Done, but feels like a clunky workaround -- to still be able to use the scroll box, needed to stick w/ `kableExtra`...but updating a single cell requires `kableExtra::cell_spec()` to mutate a cell *prior* to `kable`. Not thrilled & will work on something more reproducible, but in the meantime [here's how](https://github.com/UCSB-MEDS/UCSB-MEDS.github.io/blob/948f37c5b20a948e14a5e8f721dd7d3c40f90d3b/courses.Rmd#L50-L54) I updated individual table cells with linked text.

**Comment by allisonhorst** [comment]
Once we have all course websites we can do something nicer [like this](https://stackoverflow.com/a/48512819).

**Comment by jamiecmontgomery** [comment]
I agree, maybe we don't need the scroll boxes, and we can just do a nicely formatted table.
